packages/Tag/src/Tag.tsx

```diff
@@ -53,7 +53,10 @@ export interface TagProps extends React.ComponentProps<"div"> {
     hasError?: boolean;
     /** Callback to execute on remove tag */
     onRemove?: (id: string) => void;
-    /** Render rounded corners */
+    /**
+     * Render rounded corners
+     * After rebrand this will be deprecated and rounded will be true by default
```

Comment:
should this be flagged with a TODO or something? or have we logged a jira card to remind us of this?

Reply:
Good point, a [story](https://gsoftdev.atlassian.net/browse/IGL-89) in jira has been created for this.